gammapy/utils/tests/test_random.py

```diff
@@ -2,14 +2,38 @@
 from __future__ import print_function, division
 import numpy as np
 from numpy.testing import assert_allclose
+from astropy.coordinates import Angle
+from astropy.tests.helper import assert_quantity_allclose
 from ..random import sample_sphere, sample_powerlaw, sample_sphere_distance
 
 
 def test_sample_sphere():
+
+    # test general case
     np.random.seed(0)
     lon, lat = sample_sphere(size=2)
-    assert_allclose(lon, [3.44829694, 4.49366732])
-    assert_allclose(lat, [0.20700192, 0.08988736])
+    assert_quantity_allclose(lon, Angle([3.44829694, 4.49366732], 'radian'))
+    assert_quantity_allclose(lat, Angle([0.20700192, 0.08988736], 'radian'))
+
+    # test specify a limited range
+    lon_min = Angle(40., 'degree')
+    lon_max = Angle(45., 'degree')
+    lat_min = Angle(10., 'degree')
+    lat_max = Angle(15., 'degree')
+    lon, lat = sample_sphere(size=10,
+                             lon_range=Angle([lon_min, lon_max]),
+                             lat_range=Angle([lat_min, lat_max]))
+    assert (lon_min <= lon).all() & (lon < lon_max).all()
```

Comment:
I find this a bit more readable (avoids the duplicated `.all()`:
```python
assert np.alltrue(lon_min <= lon) & (lon < lon_max))
```
I'm not sure if either of these spellings is the best way to write this assert (i.e. short and easy to read, but still give a good message when it fails).

There's also [numpy.testing.assert_array_less](http://docs.scipy.org/doc/numpy/reference/generated/numpy.testing.assert_array_less.html), but I haven't used it.

So please take 10 minutes to thing / play with this some more, to make sure we have a good pattern for such asserts (we'll probably have 100s once we start adding lots of tests for Gammapy).
But in the end, yes, this is not very important and whatever you put, I'll merge.